src/scripts.c

```diff
@@ -58,10 +60,15 @@ static void update_triggers(void)
 		ret = system("/usr/bin/systemctl daemon-reload");
 	}
 
-	ret = system("/usr/bin/systemctl --no-block restart update-triggers.target");
+	if (!block) {
+		string_or_die(&block_flag, "--no-block");
+	}
+
+	string_or_die(&restart_cmd, "/usr/bin/systemctl restart %s update-triggers.target", block_flag);
```

Comment:
--no-block must come _before_ the command, aka
`+	string_or_die(&restart_cmd, "/usr/bin/systemctl %s restart update-triggers.target", block_flag);`